packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -965,6 +967,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             metadata,
         );
 
+        const shouldRunExpiry = this.runtimeOptions.gcOptions?.gcAllowed
+        && !this.runtimeOptions.gcOptions?.disableGC
+        && !this.runtimeOptions.gcOptions?.runSweep;
+
+        if(shouldRunExpiry) {
+            const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000;
```

Comment:
Can you please move this value up to some level where it is overwritable? I.e. it feels like it should be
- property of runtime options that client can overwrite (possibly with some default value)
- I assume we can't allow it to change through lifetime of a file, so it current value should be captured at file creation and recorded as file metadata and read back by runtime and used for every file.

That way we can change it over time (for newly created files only!). I.e., if we learn that users complain it's too short or two long, we at least have a mechanism to change it over time.

Reply:
Also this value will need to be exposed to other layers, such that we can do proper math (30 days + 2 days for snapshot caching = 32 days before anything can be swept)